recipes/erroranalysis/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "erroranalysis" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/erroranalysis-{{ version }}.tar.gz
+  sha256: e96b19677f48df248a223d4ece481018a73e1e055632f255c73f0699822a89a6
```

Have a try with this. This also should avoid custom build scripts and enable the use of `pip check` again.
```suggestion
  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/erroranalysis-{{ version }}.tar.gz
    sha256: e96b19677f48df248a223d4ece481018a73e1e055632f255c73f0699822a89a6
  - url: https://raw.githubusercontent.com/microsoft/responsible-ai-toolbox/{{ version }}/erroranalysis/requirements.txt
    sha256: …
    fn: requirements.txt
```

---

You should be able to do the same with the LICENSE.

---

Nice idea! The problem is that the repository releases and the PyPI package released don't align. That's because they're using a mono-repo approach, where multiple packages are versioned in the same repository.

E.g. `erroranalysis`'s latest release is `0.3.2` but the latest tag in `responsible-ai-toolbox` is `0.19.0`, see: https://github.com/microsoft/responsible-ai-toolbox/tags

So I don't think this is going to work here.